ajax/libs/repo.js/package.json

```diff
@@ -10,5 +10,6 @@
   },
   "keywords": [
     "repo.js"
-  ]
+  ],
+  "license": "MIT"
```

Comment:
@ralphchung Thank you for your contribution.
The license part need to be modified.
Form `"license": "MIT"` to 
```js
   "licenses": [
     "MIT",
     "GPL-2.0"
   ]
```
Because the source code mention the licenses part in here https://github.com/darcyclarke/Repo.js/blob/master/repo.js#L4-L7

And the commit message not `Update the license of repo.js, cc #11931` 
We add this field so let's use `Add the licenses of repo.js, cc #11931`

Thank you.

Reply:
I see!!. Thanks for the lesson.

Reply:
So, could you modify this?
If everything is OK, I can help you squash the commits to one commit. 😃 
Thanks.

Reply:
I tried to follow the instructions you provided, but I am not quite sure that I did the right thing.
Could you take a look at it?
Thank you very much!!!!